Test suite doesn't compile

I think it is missing some modules which should be listed in `other-modules`.

```
s.hs, dist/build/tests/tests-tmp/NLP/Extraction/ParsecTests.o )
[12 of 12] Compiling Main             ( tests/src/Main.hs, dist/build/tests/tests-tmp/Main.o )

tests/src/Main.hs:24:1: error:
    Could not find module ‘NLP.Corpora.BrownTests’
    Perhaps you meant NLP.Corpora.Brown (from chatter-0.9.1.0)
    Use -v (or `:set -v` in ghci) to see a list of the files searched for.
   |
24 | import qualified NLP.Corpora.BrownTests as Brown
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

tests/src/Main.hs:25:1: error:
    Could not find module ‘NLP.Corpora.ConllTests’
    Perhaps you meant NLP.Corpora.Conll (from chatter-0.9.1.0)
    Use -v (or `:set -v` in ghci) to see a list of the files searched for.
   |
25 | import qualified NLP.Corpora.ConllTests as Conll
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

tests/src/Main.hs:29:1: error:
    Could not find module ‘NLP.POS.LiteralTaggerTests’
    Perhaps you meant NLP.POS.LiteralTagger (from chatter-0.9.1.0)
    Use -v (or `:set -v` in ghci) to see a list of the files searched for.
   |
29 | import qualified NLP.POS.LiteralTaggerTests as LT
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

tests/src/Main.hs:33:1: error:
    Could not find module ‘NLP.Types.IOBTests’
    Perhaps you meant NLP.Types.IOB (from chatter-0.9.1.0)
    Use -v (or `:set -v` in ghci) to see a list of the files searched for.
   |
33 | import qualified NLP.Types.IOBTests as IOB
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

tests/src/Main.hs:34:1: error:
    Could not find module ‘NLP.Types.TreeTests’
    Perhaps you meant NLP.Types.Tree (from chatter-0.9.1.0)
    Use -v (or `:set -v` in ghci) to see a list of the files searched for.
   |
34 | import qualified NLP.Types.TreeTests as Tree
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

tests/src/Main.hs:35:1: error:
    Could not find module ‘NLP.Chunk.AvgPerceptronChunkerTests’
    Perhaps you meant
      NLP.Chunk.AvgPerceptronChunker (from chatter-0.9.1.0)
    Use -v (or `:set -v` in ghci) to see a list of the files searched for.
   |
35 | import qualified NLP.Chunk.AvgPerceptronChunkerTests as APC
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

--- a/chatter.cabal
+++ b/chatter.cabal
@@ -77,7 +77,7 @@ Library
                      containers     >= 0.5.0.0,
                      random-shuffle >= 0.0.4,
                      MonadRandom    >= 0.1.2,
-                     cereal         >= 0.4.0.1 && <= 0.5.7.0,
+                     cereal         >= 0.4.0.1 && <= 0.5.9.0,
                      cereal-text    >= 0.1 && < 0.2,
                      fullstop       >= 0.1.3.1,
                      bytestring     >= 0.10.0.0,
@@ -111,7 +111,7 @@ Executable tagPOS
                      text >= 0.11.3.0,
                      base >= 4.6 && <= 6,
                      bytestring >= 0.10.0.0,
-                     cereal >= 0.4.0.1 && <= 0.5.7.0
+                     cereal >= 0.4.0.1 && <= 0.5.9.0
 
    ghc-options:      -Wall -main-is Tagger -rtsopts
 
@@ -125,7 +125,7 @@ Executable trainPOS
                      text >= 0.11.3.0,
                      base >= 4.6 && <= 6,
                      bytestring >= 0.10.0.0,
-                     cereal >= 0.4.0.1 && <= 0.5.7.0,
+                     cereal >= 0.4.0.1 && <= 0.5.9.0,
                      containers >= 0.5.0.0
 
    ghc-options:      -Wall -main-is POSTrainer -rtsopts
@@ -140,7 +140,7 @@ Executable trainChunker
                      text >= 0.11.3.0,
                      base >= 4.6 && <= 6,
                      bytestring >= 0.10.0.0,
-                     cereal >= 0.4.0.1 && <= 0.5.7.0,
+                     cereal >= 0.4.0.1 && <= 0.5.9.0,
                      containers >= 0.5.0.0
 
    ghc-options:      -Wall -main-is ChunkTrainer -rtsopts
@@ -155,7 +155,7 @@ Executable trainNER
                      text >= 0.11.3.0,
                      base >= 4.6 && <= 6,
                      bytestring >= 0.10.0.0,
-                     cereal >= 0.4.0.1 && <= 0.5.7.0,
+                     cereal >= 0.4.0.1 && <= 0.5.9.0,
                      containers >= 0.5.0.0
 
    ghc-options:      -Wall -main-is NERTrainer -rtsopts
@@ -171,7 +171,7 @@ Executable eval
                      text >= 0.11.3.0,
                      base >= 4.6 && <= 6,
                      bytestring >= 0.10.0.0,
-                     cereal >= 0.4.0.1 && <= 0.5.7.0,
+                     cereal >= 0.4.0.1 && <= 0.5.9.0,
                      containers >= 0.5.0.0
 
    ghc-options:      -Wall -main-is Evaluate -rtsopts
@@ -225,7 +225,7 @@ test-suite tests
                      tokenize,
                      QuickCheck,
                      filepath,
-                     cereal >= 0.4.0.1 && <= 0.5.7.0,
+                     cereal >= 0.4.0.1 && <= 0.5.9.0,
                      quickcheck-instances,
                      containers,
                      tasty,

--- a/chatter.cabal
+++ b/chatter.cabal
@@ -207,11 +207,18 @@ test-suite tests
 
    Other-modules:    AvgPerceptronTests
                      BackoffTaggerTests
+                     NLP.Chunk.AvgPerceptronChunkerTests
+                     NLP.Chunk.AvgPerceptronChunkerTests
+                     NLP.Corpora.BrownTests
+                     NLP.Corpora.ConllTests
                      NLP.Similarity.VectorSimTests
                      NLP.POSTests
+                     NLP.POS.LiteralTaggerTests
                      NLP.POS.UnambiguousTaggerTests
                      NLP.Extraction.ParsecTests
                      NLP.TypesTests
+                     NLP.Types.IOBTests
+                     NLP.Types.TreeTests
                      Data.DefaultMapTests
                      Corpora
                      TestUtils

